lib/v2/screens/sign_up/viewmodels/signup_bloc.dart

```diff
@@ -85,6 +85,17 @@ class SignupBloc extends Bloc<SignupEvent, SignupState> {
       );
       Result result = await _claimInviteUseCase.unpackLink(event.scannedLink);
       yield ClaimInviteMapper().mapInviteMnemonicToState(state, result);
+
+      if (state.claimInviteState.inviteMnemonic != null) {
+        yield state.copyWith(
+          claimInviteState: state.claimInviteState.copyWith(
+            pageCommand: StopScan(),
+          ),
+        );
+        Result result = await _claimInviteUseCase.validateInviteCode(state.claimInviteState.inviteMnemonic!);
```

Comment:
ohh i see the call to back-end is made again to verify

Reply:
yes ;)